bgpd/bgp_label.h

```diff
@@ -58,7 +58,7 @@ static inline int bgp_is_withdraw_label(mpls_label_t *label)
 
 	/* The check on pkt[2] for 0x00 or 0x02 is in case bgp_set_valid_label()
 	 * was called on the withdraw label */
-	if ((pkt[0] == 0x80) && (pkt[1] == 0x00)
+	if (((pkt[0] == 0x80) || (pkt[0]) == 0x00) && (pkt[1] == 0x00)
```

Comment:
unknowingly are we enabling 0x000002 as a valid withdraw label (which is not recommended as per RFC8277/RFC3107)  ?

Reply:
I think this works as a marker to identify quickly just if the label is valid?

Reply:
The fix probably is good according to RFCs, but this `(pkt[0]) == 0x00` looks weird. Can you fix this?

Reply:
> The fix probably is good according to RFCs, but this `(pkt[0]) == 0x00` looks weird. Can you fix this?

Label no 0x000000 is a valid one as per the RFC. so according to me it should not be removed in order to be compatible with stacks sending 0x000000 instead of 0x800000.

Reply:
I meant (pkt[0]) in parenthessis.